cmd/blob-csi.go

```diff
@@ -455,7 +455,13 @@ func createPVC(client *kubernetes.Clientset, pvc *corev1.PersistentVolumeClaim)
 	)
 }
 
-func getBlobClient(client *kubernetes.Clientset, containerConfig AzureContainerConfig) (azblob.ServiceClient, error) {
+func handleError(err error) {
```

Comment:
This method does the logging for the error, but then the method will continue to process the request, causing more errors and then the method will return to the caller with a nil indicating there were no errors? I think the design expects any errors to stop processing and return it to the calling method, where it will stop the request (i.e. here https://github.com/StatCan/aaw-kubeflow-profiles-controller/blob/e19e0102fab2476730901587caacef666ead74c0/cmd/blob-csi.go#L650C4-L650C4)

Reply:
### My thinking:
If the blobclient fails to create for whatever reason, there's no point running anything else since that'll mostlikely be dependent on a blobclient to make requests to the AZBlob API. 
I believe handleError will force the program to exit. We can probably get rid of L650-L652 as they're unreachable anyways. We can probably also get rid of the error return value in `getBlobClient` since it's dead code, it's set to nil anyways.

Reply:
If we'd like exectution to resume for whatever reason, I can just replace log.fatal with log.panic

Reply:
I'm not very familiar with the whole app, but what I see around L650-L652 is that there is a loop to load all blob clients in the aaw config, if any of those fail now the whole program execution is killed and none will be loaded? 

Previously it just returned any errors that were encountered right away, then L650-L652 logged the panic and it continued to process the rest of the config so other containers can still be mounted?

Overall though I would say that if the method is expected to return an err then it should do that, or the method should be changed it indicate it doesn't return any errors.